frontend/common/stores/organisation-store.js

```diff
@@ -125,7 +126,7 @@ const controller = {
       })
   },
   getOrganisation: (id, force) => {
-    if (id !== store.id || force) {
+    if (`${id}` !== `${store.id}` || force) {
```

Comment:
Protects against refetching organisation where the only difference was id vs "id"